ocaml/quicktest/quicktest_vm_import_export.ml

```diff
@@ -87,22 +85,22 @@ let import_export_test session_id sr_info =
              then Alcotest.fail
                  (Printf.sprintf "Device %s varies in VDIness (original = %s; new = %s)"
                     all.API.vBD_userdevice
-                    (Client.Client.VDI.get_uuid !rpc session_id orig_vbd.API.vBD_VDI)
-                    (Client.Client.VDI.get_uuid !rpc session_id all.API.vBD_VDI)
+                    (Client.Client.VDI.get_uuid rpc session_id orig_vbd.API.vBD_VDI)
+                    (Client.Client.VDI.get_uuid rpc session_id all.API.vBD_VDI)
                  )
            | "3" ->
              (* VDI should be different *)
              if all.API.vBD_VDI = orig_vbd.API.vBD_VDI
              then Alcotest.fail (Printf.sprintf "Device %s should not vary in VDIness" all.API.vBD_userdevice)
```

Comment:
I still don't understand what VDIness is???

Reply:
I think VDIness means that the VDI of one VBD is the same as the VDI of another